client/src/components/Form/FormListElementOperations.vue

```diff
@@ -0,0 +1,70 @@
+<script setup lang="ts">
+import { library } from "@fortawesome/fontawesome-svg-core";
+import { faCaretDown, faCaretUp, faTrashAlt } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/vue-fontawesome";
+import { BButton, BButtonGroup } from "bootstrap-vue";
+
+// @ts-ignore: bad library types
+library.add(faTrashAlt, faCaretUp, faCaretDown);
+
+interface Props {
+    index: number;
+    numElements: number;
+    upButtonId: string;
+    downButtonId: string;
+    canDelete: boolean;
+    deleteTooltip: string;
+}
+
+defineProps<Props>();
+
+const emit = defineEmits<{
+    (e: "delete"): void;
+    (e: "swap-up"): void;
+    (e: "swap-down"): void;
+}>();
+</script>
+
+<template>
+    <span class="float-right">
+        <BButtonGroup>
+            <BButton
+                :id="upButtonId"
+                v-b-tooltip.hover.bottom
+                :disabled="index == 0"
+                title="move up"
+                role="button"
+                variant="link"
+                size="sm"
+                class="ml-0"
+                @click="() => emit('swap-up')">
+                <FontAwesomeIcon icon="caret-up" />
```

Comment:
This will correctly reference and bundle the icon.

```suggestion
                <FontAwesomeIcon :icon="faCaretUp" />
```